queenbee/base/request.py

```diff
@@ -1,19 +1,34 @@
+import pathlib
+import os
 from urllib import request
 from typing import Union
-
 from .basemodel import BaseModel
 
 USER_AGENT_STRING = 'Queenbee'
 
 
+def get_uri(url):
+    """Resolve uri for urls and local files."""
+
+    if url.startswith('file:///'):
+        return url.replace('\\', '/')
+
+    try:
+        uri = pathlib.Path(os.path.abspath(url)).as_uri()
```

Comment:
Why are we trying to resolve an http url as an absolute path? Lines 13-14 address the case where a url is for a local path so maybe we should make that one an `abspath`.

What happens if we keep this line is that all the `http` urls are appended with a local file path like so:

```
file:///root/projects/ladybug-tools/queenbee/https%3A/api.pollination.cloud/registries/antoinedao
```

Reply:
Good catch! I will fix this.

Reply:
Still not sure this is resolved. From my understanding a `url` should only start with either: `file://`, `http://` or `https://`. Therefore lines 18/24 just look like a no-op to me.

Reply:
You hit line 18-24 the first time that you add a local repository. In that case `url` is a file but doesn't start with a `file:///`.

`queenbee repo add honeybee-radiance usr/path/to/local/repo`

Reply:
Oh! Great catch! 😅